Postman/Postman.js

```diff
@@ -107,111 +101,108 @@ Seriesコレクションから新刊のBookListドキュメントを抽出する
 @param { FUnction } done
 @return { Error | Array } 新刊のBookListコレクション
 **/
-Postman.prototype.inspectSeries = function(seriesItems, done){
-	var i, seriesItem, newReleasies = [];
-	for (i = 0; i < seriesItems.length; i++) {
-		seriesItem = seriesItems[i];
-		newReleasies = this._diffItems(seriesItem.recentContains, seriesItem.currentContains);
-	}
-	if(newReleasies.length === 0){
-		return done({
-			"status": 'not-sent',
-			"message": "通知するシリーズ書籍がありません"
-		});
-	}
-	done(null, newReleasies);
+Postman.prototype.inspectSeries = function inspectSeries(seriesItems, done) {
+  let index;
+  let seriesItem;
+  let newReleasies = [];
+  for (index = 0; index < seriesItems.length; index++) {
+    seriesItem = seriesItems[index];
+    newReleasies = this._diffItems(seriesItem.recentContains, seriesItem.currentContains);
+  }
+
+  if (newReleasies.length === 0) {
+    return done({
+      'status': 'not-sent',
+      'message': '通知するシリーズ書籍がありません',
+    });
+  }
+  done(null, newReleasies);
 };
 
 
-/**
-**/
-Postman.prototype._diffItems = function(srcArray, criteriaArray){
-	var diffArray = _.filter(srcArray, function(obj){
-		return !_.findWhere(criteriaArray, obj);
-	});
-	return diffArray;
+Postman.prototype._diffItems = function _diffItems(srcArray, criteriaArray) {
+  const diffArray = _.filter(srcArray, (obj)=> {
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).

*  'arrow function syntax (=>)' is only available in ES6 (use esnext option).